examples/live/bybit/bybit_request_custom_endpoint.py

```diff
@@ -77,7 +75,7 @@ def on_start(self):
     def send_tickers_request(self, time_event: TimeEvent) -> None:
         data_type = DataType(
             BybitTickerData,
-            metadata={"symbol": self.instrument_id.symbol},
+            metadata={"symbol": self.cconfig.instrument_id.symbol},
```

Comment:
`self.config`

Reply:
Typo fixed.

Thank you very much Chris for all your support.
I learnt a lot during this first PR 👍